src/server.h

```diff
@@ -2595,10 +2594,20 @@ int serverCommunicateSystemd(const char *sd_notify_msg);
 void serverSetCpuAffinity(const char *cpulist);
 void dictVanillaFree(dict *d, void *val);
 
+/* ERROR STATS constants */
+
+/* Once the errors RAX reaches this limit, instead of tracking custom
+ * errors (e.g. LUA), we track the error under the prefix below. */
+#define ERRORSTATS_LIMIT 128
```

Comment:
Do we want to make this configurable or is this sufficient?

Reply:
Sufficient IMO. I want to avoid configs if we don't have an actual use case for the config.